components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,767 @@
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-2/12">
+                            <span className="my-auto truncate text-gray-500 overflow-ellipsis">{slots.filter(s => s.teamSubscription.id === sub.id).length || "–"}</span>
+                            <span className="text-sm my-auto text-gray-400">Members</span>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-5/12">
+                            <span className="my-auto truncate text-gray-500 overflow-ellipsis">{"–"}</span>
+                            <span className="text-sm my-auto text-gray-400">Next Biling Cycle</span>
+                        </div>
+                        <div className="my-auto flex w-1/12 pl-8">
+                            <div className="self-center hover:bg-gray-200 rounded-md cursor-pointer w-8">
+                                <ContextMenu menuEntries={subscriptionMenu(sub)}>
+                                    <img className="w-8 h-8 p-1" src={ThreeDots} alt="Actions" />
+                                </ContextMenu>
+                            </div>
+                        </div>
+                    </div>
+                ))}
+            </div>
+        )}
+
+
+    </div>);
+}
+
+function InviteMembersModal(props: {
+    sub: TeamSubscription,
+    onClose: () => void
+}) {
+
+    const getInviteURL = () => {
+        const link = new URL(window.location.href);
+        link.pathname = '/subscription/'
+        link.search = '?teamid=' + props.sub.id;
+        return link.href;
+    }
+
+    const copyToClipboard = (text: string) => {
+        const el = document.createElement("textarea");
+        el.value = text;
+        document.body.appendChild(el);
+        el.select();
+        try {
+            document.execCommand("copy");
+        } finally {
+            document.body.removeChild(el);
+        }
+    };
+
+    return (<Modal visible={true} onClose={props.onClose}>
+        <h3 className="pb-2">Invite Members</h3>
+        <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-4 space-y-2">
+            <p className="pb-4 text-gray-500 text-base">Invite members to the team using the URL below.</p>
+
+            <div className="flex flex-col space-y-2">
+                <label htmlFor="inviteUrl" className="font-medium">Invite URL</label>
+                <div className="w-full relative">
+                    <input name="inviteUrl" disabled={true} readOnly={true} type="text" value={getInviteURL()} className="rounded-md w-full truncate pr-8" />
+                    <div className="cursor-pointer" onClick={() => copyToClipboard(getInviteURL())}>
+                        <img src={copy} title="Copy Invite URL" className="absolute top-1/3 right-3" />
+                    </div>
+                </div>
+                <p className="pb-4 text-gray-500 text-xs">Use this URL to join this team</p>
```

Comment:
**suggestion**: Let's try to not use anything below `text-sm` across the product.

```suggestion
                <p className="pb-4 text-gray-500 text-sm">Use this URL to join this team</p>
```